### ISSUE TITLE
"Cannot find package 'glob'" when running npx create-puck-app my-app

### ISSUE THREAD

**Issue by maantjemol** [measuredco/puck#38]
Hi!
What an amazing project! I was trying to get it to work locally but it seems like the recipe doesn't work on my machine. I get this error when I run it:
```
> npx create-puck-app my-app

node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'glob' imported from /Users/maantje/.npm/_npx/039b3c3c47fc2298/node_modules/create-puck-app/index.js
    at new NodeError (node:internal/errors:405:5)
    at packageResolve (node:internal/modules/esm/resolve:781:9)
    at moduleResolve (node:internal/modules/esm/resolve:830:20)
    at defaultResolve (node:internal/modules/esm/resolve:1035:11)
    at DefaultModuleLoader.resolve (node:internal/modules/esm/loader:251:12)
    at DefaultModuleLoader.getModuleJob (node:internal/modules/esm/loader:140:32)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:33)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v20.4.0
```

I'm using Node v20.4.0, I also tried with pnpm and got the same error. I sadly don't know enough about npm to fix this.

**Comment by gajraj-gan** [comment]
+1

**Comment by kumarajay0412** [comment]
+1

**Comment by chrisvxd** [comment]
Thanks for the report! Can confirm I'm able to reproduce. Investigating now.

**Comment by chrisvxd** [comment]
This has been released in `v0.6.1`.

`npx create-puck-app my-app` should work now.
